knowledge-base/clj-holmes/clj-holmes-action/action-security.yml

```diff
@@ -0,0 +1,7 @@
+name: 'clj-holmes (Clojure)'
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  permissions:
+    contents: read 
+    contents-reason: to download rules for private repo
+    #Reference: https://github.com/clj-holmes/clj-holmes/blob/1918dbb5d16a8e37461a4233891a6f9f84229c63/src/clj_holmes/rules/wagon/github.clj#L14
```

Comment:
@Devils-Knight is this downloading for the repo on which action is run or a different repo. If different repo, it does not need `contents:read`

Reply:
@varunsh-coder Rules are downloaded for the repo on which the action is run and the rules are to be fetched from a different repo.